taar/adapters/dynamo.py

```diff
@@ -0,0 +1,53 @@
+import boto3
+import json
+
+
+class ProfileController(object):
+    """
+    This class
+    """
+
+    def __init__(self, region_name, table_name):
+        """
+        Configure access to the DynamoDB isntance
+        """
+        self._ddb = boto3.resource('dynamodb', region_name=region_name)
+        self._table = self._ddb.Table(table_name)
```

Comment:
Can any of these two lines fail?

Reply:
Yes, but that should be fine.  Failure to bind a connection in the constructor shouldn't be masked with an exception, I'd rather just see the object construction fail entirely.  In practice, the taar-api will just give the user a 500 error if the RecommendationManager and it's dependencies including the ProfileController cannot be created.